content/3_Silver/Binary_Search.mdx

```diff
@@ -1198,27 +1198,26 @@ def binary_search(x: int, a: list):
 </PySection>
 </LanguageSection>
 
-
     <Quiz.Answer>
-      $\mathcal{O}(\log n)$ / $\Omega(\log n)$
+      $\mathcal{O}(\log n)$ / $\mathcal{O}(\log n)$
```

Comment:
This should use $\Theta$ rather than $\mathcal O$ for all quiz answers

Reply:
alright